sdk/provisioning/Azure.Provisioning.Storage/tests/BasicStorageTests.cs

```diff
@@ -0,0 +1,389 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Provisioning.Authorization;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Roles;
+using Azure.Provisioning.Tests;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.Storage.Tests;
+
+public class BasicStorageTests(bool async)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true /**/)
+{
+    [RecordedTest]
+    public async Task CreateDefault()
+    {
+        await using Trycep test = CreateBicepTest();
+        await test.Define(StorageResources.CreateAccount("storage"))
+        .Compare(
+            """
+            resource storage 'Microsoft.Storage/storageAccounts@2023-01-01' = {
+                name: take('storage${uniqueString(resourceGroup().id)}', 24)
+                kind: 'StorageV2'
+                location: resourceGroup().location
+                sku: {
+                    name: 'Standard_LRS'
+                }
+                properties: {
+                    allowBlobPublicAccess: false
+                    isHnsEnabled: true
+                }
+            }
+            """)
+        .Lint()
+        .ValidateAndDeployAsync();
+    }
```

Comment:
Or you can use a convenience helper like `StorageResources.CreateAccount("storage")` to give you a bunch of reasonable defaults.